packages/webview_flutter_lwe/lib/src/lwe_webview_controller.dart

```diff
@@ -62,14 +62,6 @@ class LweWebViewController extends PlatformWebViewController {
           'LoadRequestParams#uri is required to have a scheme.');
     }
 
-    if (params.headers.isNotEmpty) {
-      throw ArgumentError('LoadRequestParams#headers is not supported.');
-    }
-
-    if (params.body != null) {
-      throw ArgumentError('LoadRequestParams#body is not supported.');
-    }
```

Comment:
Just curious: is there a reason to skip this check? Are there cases when the body and header are empty?

Reply:
The params.headers and params.body variables are used in post method, and we already had UnimplementedError() to handle those exceptions. So I thought the conditions were redundant.